prac_06/guitar.py

```diff
@@ -0,0 +1,30 @@
+"""
+guitar.py
+Estimate - 45 minutes
+Actual - 30 minutes
+"""
+
+CURRENT_YEAR = 2023
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Represent a guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a guitar instance."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return name, year and cost."""
+        return f"{self.name} ({self.year}) : ${self.cost:.2f}"
```

Comment:
adding a comma to ${self.cost:,.2f} means it'll print the cost as $16,035.40 instead of $16035.40. Small detail, not really important